pkg/tnf/interactive/spawner.go

```diff
@@ -272,6 +281,22 @@ func NewGoExpectSpawner() *GoExpectSpawner {
 	return &GoExpectSpawner{}
 }
 
+// LogStdErr logs stderr output to logger (warning)
+func LogStdErr(cmdLine string, stderrpipe io.Reader) {
```

Comment:
do we need to export this function outside of spawner ? I think using "logStdErr" instead of "LogStdErr" is preferred unless we want to use it somewhere else ?

Reply:
fixed